package.json

```
@@ -16,24 +16,45 @@
     "url": "https://github.com/Financial-Times/origami-registry-ui.git"
   },
   "engines": {
-    "node": "^12.16.1",
-    "npm": "^6.13.0"
+    "node": "^14.16.1",
+    "npm": "^7.11.2"
   },
   "main": "./lib/service.js",
   "scripts": {
     "start": "heroku-node-settings index.js",
-    "build:css": "mkdir -p public && sass src/main.scss  --style=compressed --load-path=bower_components --load-path=. | postcss --use autoprefixer --output public/main.css --map",
-    "build:js": "mkdir -p public && oax ./src/main.js --for-bower | swc -f - -C minify > public/main.js",
+    "build:css": "mkdir -p public && sass src/main.scss  --style=compressed --load-path=node_modules --load-path=node_modules/@financial-times --load-path=. | postcss --use autoprefixer --output public/main.css --map",
+    "build:js": "mkdir -p public && oax ./src/main.js | swc -f - -C minify > public/main.js",
     "build": "npm-run-all --parallel build:js build:css",
     "watch:css": "nodemon --watch src -e scss -x 'npm run build:css'",
     "watch:js": "nodemon --watch src -e js -x 'npm run build:js'",
     "watch": "npm-run-all --parallel watch:js watch:css",
-    "heroku-postbuild": "bower install && npm run build",
+    "heroku-postbuild": "npm install && npm run build",
```

We don't need to have the `npm install` part here, Heroku does a full install (dev-dependencies) before running the `heroku-postbuild` script and prunes the dependencies after the script finishes -- https://devcenter.heroku.com/articles/nodejs-support#package-installation

```suggestion
    "heroku-postbuild": "npm run build",
```